src/components/cylc/GraphSubgraph.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <g class="c-graph-subgraph">
+    <rect
+      :width="subgraph.width"
+      :height="subgraph.height"
+      :x="subgraph.x"
+      :y="subgraph.y"
+      rx="50"
+      ry="50"
+      fill="none"
+      stroke-width="8px"
+      stroke="grey"
+      stroke-dasharray="50 50"
+      />
+    <text
+      :x="labelXPosition"
+      :y="labelYPosition"
+      font-family="Roboto"
+      alignment-baseline="middle" text-anchor="middle"
+      font-size="70px"
```

Comment:
The fontsize here *should* match the "70px" defined in the graphviz document. But, unfortunately, the font family does not match, because different fonts may render different sizes (for the same `px` value) this value is a bit off. The result is that the label may overflow the box:

![Screenshot from 2024-05-14 09-55-37](https://github.com/cylc/cylc-ui/assets/16705946/95be5397-5162-427a-bac8-ae3bf7672704)

To correct this, we could define`fontfamily="roberto"` in the dot code. This works a charm locally, but sadly, it doesn't appear to work in the browser. I expect graphviz doesn't have access to this font so is falling back to something else.

So I think the best solution is to apply a fudge factor here to ensure the text fits, here's the number I came up with by trial and error:

```suggestion
      font-size="60px"
```

Which results in:

![Screenshot from 2024-05-14 10-10-39](https://github.com/cylc/cylc-ui/assets/16705946/385b1b3d-cad8-43ac-bc2b-01d381d53c61)